tests/core/ipfs/Qm/NQ/euwMDGJ7UiLaRjwzAoekcaKLp9TjiqeFdovj3syN1n/QmNQeuwMDGJ7UiLaRjwzAoekcaKLp9TjiqeFdovj3syN1n

```diff
@@ -0,0 +1,23 @@
+pragma solidity ^0.4.24;
```

Comment:
Looks like fixtures that should be zipped up to prevent accidentally modifying them in-place?